packages/plugin-button/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "wix-rich-content-plugin-button",
+  "version": "2.0.4",
+  "author": {
+    "name": "Bashar Qaddumi",
+    "email": "bqaddumi@asaltech.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wix-incubator/rich-content.git"
+  },
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/wix-incubator/rich-content/issues"
+  },
+  "homepage": "https://github.com/wix-incubator/rich-content/tree/master/packages/plugin-button",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/module.cjs.js",
+  "module": "dist/module.js",
+  "style": "dist/styles.min.scss",
+  "scripts": {
+    "clean": "rm -rf dist/",
+    "statics": "mkdir -p dist && cp -R statics dist",
+    "prebuild": "npm run statics",
+    "build": "rollup --environment MODULE_NAME:plugin-button -c ../../rollup.config.js",
+    "build:dev": "NODE_ENV=development npm run build",
+    "build:analyze": "npm run build -- --environment MODULE_ANALYZE && open stats.html",
+    "watch": "NODE_ENV=development MODULE_WATCH=1 npm run build -- --watch",
+    "lint": "eslint 'src/**/*.{js,jsx}'"
+  },
+  "peerDependencies": {
+    "@babel/runtime": "7.2.0",
+    "@wix/draft-js": "0.10.272",
+    "classnames": "^2.0.0",
+    "lodash": "^4.0.0",
+    "prop-types": "^15.0.0",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2"
+  },
+  "devDependencies": {
+    "@babel/runtime": "7.2.0",
+    "@wix/draft-js": "0.10.272",
+    "classnames": "^2.2.6",
+    "eslint": "^4.19.1",
+    "lodash": "^4.17.10",
+    "prop-types": "^15.6.2",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2",
+    "rollup": "^1.1.0",
+    "eslint-import-resolver-lerna": "^1.0.0",
+    "eslint-plugin-import": "2.8.0"
+  },
+  "dependencies": {
+    "react-color": "^2.14.1",
+    "react-custom-scrollbars": "^4.2.1",
+    "wix-rich-content-common": "^2.0.4"
```

bump to latest version
```suggestion
    "wix-rich-content-common": "^3.1.1"
```